episodes/11-configuring-github.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Configuring GitHub
+teaching: 15
+exercises: 0
+---
+
+::::::::::::::::::::::::::::::::::::::: objectives
+
+- Configure your GitHub profile and settings.
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::: questions
+
+- How do I change my notification preferences?
```

Comment:
I think you are answering more questions in this like 'How do I configure my optional settings?' 'How do I change the visibility of my organisation membership?'